api/v1alpha1/openapi.yaml

```diff
@@ -28,7 +28,22 @@ tags:
     description: Operations on Repository resources.
   - name: resourcesync
     description: Operations on ResourceSync resources.
+  - name: version
+    description: Operations for receiving service version.
 paths:
+  /api/version:
+    get:
+      tags:
+        - version
+      description: Get service version.
+      operationId: version
```

Comment:
Nit: "getVersion", as this is what gets used by the generator to name handlers and so we have consistency.

Reply:
done